src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represents a tag operation
+ */
+public class Tagging {
+
+    private final ReadOnlyPerson person;
+    private final Tag tag;
+    private final String operation;
```

Comment:
think why `String` may not be the best data type for this particular field?
what would be more appropriate?